script/constants/environments.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  DEVELOPMENT: 'development',
+  STAGING: 'staging',
+  PRODUCTION: 'production',
+  VAGOVDEV: 'vagovdev',
+  VAGOVSTAGING: 'vagovstaging',
+  PREVIEW: 'preview'
+};
```

Comment:
This was just part of the general cleanup. These values were just floating around otherwise.